awx/main/access.py

```diff
@@ -861,6 +863,61 @@ def can_unattach(self, obj, sub_obj, relationship, *args, **kwargs):
         return super(OrganizationAccess, self).can_attach(obj, sub_obj, relationship, *args, **kwargs)
 
 
+class HostMetricAccess(BaseAccess):
+    """
+    - I can see host metrics when I'm a super user or system auditor.
+    - I can delete host metrics when I'm a super user.
+    """
+
+    model = HostMetric
+
+    def get_queryset(self):
+        if self.user.is_superuser or self.user.is_system_auditor:
+            qs = self.model.objects.all()
+        else:
+            qs = self.filtered_queryset()
+        return qs
+
+    def can_read(self, obj):
+        return bool(self.user.is_superuser or self.user.is_system_auditor or (obj and obj.user == self.user))
+
+    def can_add(self, data):
+        return False  # There is no API endpoint to POST new settings.
+
+    def can_change(self, obj, data):
+        return False
+
+    def can_delete(self, obj):
+        return bool(self.user.is_superuser or (obj and obj.user == self.user))
+
+
+class HostMetricSummaryMonthlyAccess(BaseAccess):
+    """
+    - I can see host metrics when I'm a super user or system auditor.
+    """
+
+    model = HostMetricSummaryMonthly
+
+    def get_queryset(self):
+        if self.user.is_superuser or self.user.is_system_auditor:
+            qs = self.model.objects.all()
+        else:
+            qs = self.filtered_queryset()
+        return qs
+
+    def can_read(self, obj):
+        return bool(self.user.is_superuser or self.user.is_system_auditor or (obj and obj.user == self.user))
+
+    def can_add(self, data):
+        return False  # There is no API endpoint to POST new settings.
+
+    def can_change(self, obj, data):
+        return False
+
+    def can_delete(self, obj):
+        return False
```

Comment:
Because this is just superuser/auditor, and that is already handled in the permission classes, I don't think we should have access classes.

I left this comment in the other PR, and you mentioned getting a 500 error. I would rather work through that error than add dead code.

Reply:
Ok, I'm not familiar with Django yet, so I'll ask you in Slack what is and isn't  needed regarding the permissions here

Reply:
@AlanCoding I digged deeper into it. More systematic approach seems to be remove the "permission_classes" from the views. Because they are almost not used except one specific case. 
On the other side working through error means doing something like
```
if model class == 'HostMetric' or model_class == 'HostMetricSummaryMonthly':
   ...
else:
   ...
```
in User related method: [User.get_query_set()](https://github.com/ansible/awx/blob/05af2972bf3799d3059363dd721d300435ccdbb4/awx/main/access.py#L143)
that's much worse